server/data/quotes.json

```diff
@@ -1,4 +1,13 @@
-[
+[ 
+  { 
+    "sentence": "One who speaks only one language is one person, but one who speaks two languages is two people.",
```

Comment:
Hi @TaniaMalhotra! This is a duplicate, hope you can remove it before I merge :)